app/views/templates/package_action.erb

```diff
@@ -0,0 +1,52 @@
+<%#
```

Comment:
I think that's way out of scope for this PR and should be a separate issue if we do it at all. There's no connection between OS and JobTemplates exposed to the user today.